### PR TITLE
feat(CI): ensure SDR data is valid

### PR DESCRIPTION
**Pull request by revolunet**
Check if services-de-renseignements.json urls are valid (allow_failure=true)

### CODE REVIEW

File: .gitlab-ci/stages/quality.yml [hunk 1]
```diff
@@ -97,3 +97,13 @@ Test @cdtn/data:
     - yarn workspace @cdt/data test
     - yarn workspace @cdt/data check-slugs
     - yarn workspace @cdt/data...datafiller test
+
+Test @cdtn/frontend SDR data:
+  <<: *quality_stage
+  image: node:12-alpine
```
Comment (lionelB) [inline .gitlab-ci/stages/quality.yml]: does an alpine3 image coud be enough ?

Comment (revolunet) [reply .gitlab-ci/stages/quality.yml]: yes

Comment (revolunet) [reply .gitlab-ci/stages/quality.yml]: which image ?

Comment (lionelB) [reply .gitlab-ci/stages/quality.yml]: alpine:3 (without nodejs)

**Review by lionelB — APPROVED** [review]
nice addition !

**Pull request merged**